### PR TITLE
Readme: Fix test example and link to test suite

### PR DESCRIPTION
**Pull request by andys8**
**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [ ] refactoring / style
- [ ] build / chore
- [x] documentation

**Summary**

The example was using a variable that didn't exist. This PR fixes the variable.
I think the example could still be incomplete, compared to the actual tests.
I added a link to the actual test suite, because they're very helpful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,14 +109,14 @@ it('tests drag state', async () => {
...
-  const child = updatedDropzone.find(DummyChildComponent)
+  const child = dropzone.find(DummyChildComponent)
   expect(child).toHaveProp('isDragActive', true)
   expect(child).toHaveProp('isDragAccept', false)
   expect(child).toHaveProp('isDragReject', true)
 })
 ```
 
-Remember to update your mounted component before asserting any props. A complete example for this can be found in `react-dropzone`s own test suite.
+Remember to update your mounted component before asserting any props. A complete example for this can be found in `react-dropzone`s own [test suite](https://github.com/react-dropzone/react-dropzone/blob/master/src/index.spec.js).
```
Comment (rolandjitsu) [inline README.md]: I think you could use a relative link here.

Comment (andys8) [reply README.md]: No, because the content will be under https://react-dropzone.netlify.com, too, and the link would be broken.

Comment (rolandjitsu) [reply README.md]: True, I did not take that into account 👍

**Comment by rolandjitsu** [comment]
One last thing, would please make sure you commit message follows the conventional commit spec?

**Pull request merged**